cli/admin/src/main/java/io/pravega/cli/admin/segmentstore/ReadSegmentRangeCommand.java

```diff
@@ -42,29 +45,43 @@ public ReadSegmentRangeCommand(CommandArgs args) {
     }
 
     @Override
-    public void execute() throws ExecutionException, InterruptedException, TimeoutException {
-        ensureArgCount(4);
+    public void execute() throws ExecutionException, InterruptedException, TimeoutException, IOException {
+        ensureArgCount(5);
 
         final String fullyQualifiedSegmentName = getArg(0);
         final int offset = getIntArg(1);
```

Comment:
Please convert both of these into `long`. We definitely have offsets greater than 2GB and (see my comment below), the user may want to download more than 2GB at once.

Reply:
The `SegmentHelper.readSegment` call takes the offset and length as integers. While it makes sense to take them as `long` since as described below the read will be as follows (offset, length).
```
[offset, 8000000] -> [offset + 8000000, 8000000] -> [offset + 1600000, 8000000] -> ........
```

Reply:
I can add a TODO referencing this: https://github.com/pravega/pravega/issues/6183

Reply:
OK. Let's do the following:
1. Please add a TODO and reference that other ticket. And please make a note in 6183 to fix this usage as well - actually add a note to check all usages and fix them as needed.
2. Do change these (both) to `long` (including `length` as we may want to download more than 2GB at once). In your API call to `SegmentHelper` add a cast to int (`(int)`) and write your TODO there. That way when 6183 is actually fixed all we need to do is remove the cast and be done.

Reply:
Added the TODO referencing #6183 . Made them `long` and typecast to `int`. We only need to remove the cast when that fix goes in.